presto-main/src/main/java/com/facebook/presto/sql/gen/InCodeGenerator.java

```diff
@@ -30,10 +30,7 @@
 import com.google.common.collect.ImmutableListMultimap;
 import com.google.common.collect.ImmutableSet;
 
-import java.util.Collection;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Our code style doesn't allow them because they cause problems (if you star import multiple packages, then later the same class is added to both packages, your code no longer compiles).

Your IDE should have an option to turn it off (or set the threshold very high). If you use IDEA we have a code style here: https://github.com/airlift/codestyle